packages/superset-ui-core/src/chart/index.ts

```diff
@@ -18,3 +18,7 @@ export { default as ChartDataProvider } from './components/ChartDataProvider';
 export * from './types/Base';
 export * from './types/TransformFunction';
 export * from './types/QueryResponse';
+
+export { default as __hack_reexport_chart_Base } from './types/Base';
+export { default as __hack_reexport_chart_TransformFunction } from './types/TransformFunction';
+export { default as __hack_reexport_chart_QueryResponse } from './types/QueryResponse';
```

Comment:
This hack fixes script errors in dev build when re-exporting pure type files:

![image](https://user-images.githubusercontent.com/335541/108666004-08180480-748b-11eb-805e-d70ec416199f.png)

Not sure what is the root cause, but this fixes it.